### ISSUE TITLE
demo page: a simpler version 

### ISSUE THREAD

**Issue by ghost** [PicNet/tablefilter#6]
The current demo page uses JS code in order to create a random table, and has additional filters in the BODY.

It might prove to be much simpler, if there would be a live demo with the simplest HEAD, and the simplest BODY, of a standard table:
Column 1: non editable index column
Column 2: a drop-down filter
Column 3: free-text search

Below is a sample of such a table. Also attached as a full demo2.htm file

Dotan

<html xmlns="http://www.w3.org/1999/xhtml" >
<HEAD>
<META HTTP-EQUIV="CONTENT-TYPE" CONTENT="text/html">
<title>PicNet Table Filter Simple Demo</title>

<script src="http://ajax.microsoft.com/ajax/jquery/jquery-1.3.2.min.js" type="text/javascript"></script>

<script type="text/javascript" src="picnet.table.filter.min.js"></script>

<script type="text/javascript">
$(document).ready(function() {
// Initialise Plugin
var options1 = {};
$('#demotable1').tableFilter(options1);
});
</script>

<style>
* { font-family:arial; font-size:8pt;}
table, td, th { border: solid 1px silver; color:#666; padding:3px 5px 3px 5px}
th { background-color:#333; color:#fff; font-size:0.85em }
a { color:gray; }
</style>

</HEAD>
<BODY>

<TABLE id="demotable1" border=1>
<THEAD>
<TR>
<TH filter='false'>Index</TH>
<TH filter-type='ddl'>Drop-down</TH>
<TH>Free-text</TH>
</TR>
</THEAD>
<TBODY>
<TR VALIGN=TOP>
<TD>1</TD>
<TD>Item1</TD>
<TD>John</TD>
</TR>
<TR VALIGN=TOP>
<TD>2</TD>
<TD>Item2</TD>
<TD>Dean</TD>
</TR>
<TR VALIGN=TOP>
<TD>3</TD>
<TD>Item3</TD>
<TD>Henry</TD>
</TR>
<TR VALIGN=TOP>
<TD>4</TD>
<TD>Item1</TD>
<TD>Jenny</TD>
</TR>
<TR VALIGN=TOP>
<TD>5</TD>
<TD>Item2</TD>
<TD>Bella</TD>
</TR>
<TR VALIGN=TOP>
<TD>6</TD>
<TD>Item3</TD>
<TD>Jean</TD>
</TR>

</TBODY>
</TABLE>

<hr/>

This plugin was developed by <a href='mailto:guido.tapia@picnet.com.au'>Guido Tapia</a> Manager for the <a href='http://www.picnet.com.au/software_development.html'>custom software development</a> branch of <a href='http://www.picnet.com.au'>PicNet</a>.

</BODY>
</html>
